pyproject.toml

```
@@ -11,21 +11,23 @@ description = "MQT Predictor - A MQT tool for Determining Good Quantum Circuit C
 readme = "README.md"
 authors = [
     { name = "Nils Quetschlich", email = "nils.quetschlich@tum.de" },
-    { name = "Lukas Burgholzer", email = "lukas.burgholzer@jku.at"},
+    { name = "Lukas Burgholzer", email = "lukas.burgholzer@tum.de"},
 ]
 keywords = ["MQT",  "quantum computing", "compilation", "machine learning", "prediction"]
 license = { file = "LICENSE" }
 requires-python = ">=3.10"
 dynamic = ["version"]
 
 dependencies = [
-    "mqt.bench @ git+https://github.com/cda-tum/mqt-bench.git",
+    "mqt.bench>=1.1.1, <1.1.2",
+    "pytket_qiskit>=0.52.0, <0.53.0",
     "sb3_contrib>=2.0.0, <2.2.2",
     "scikit-learn>=1.4.0,<1.4.2",
     "importlib_metadata>=4.4; python_version < '3.10'",
     "importlib_resources>=5.0; python_version < '3.10'",
     "tensorboard>=2.11.0, <2.16.3",
-    "bqskit>=1.1.0, <1.1.2",
+    "bqskit>=1.1.2, <1.1.3",
+    "lark~=0.7"
```

Just out of curiosity: where is this dependency coming from and if it is really needed, could we state it similar to the previous requirements with a lower and an upper bound (if necessary)?

---

I added lark dependency since pytket is depending on the (deprecated) lark-parser~=0.7 while BQSKit has upgraded to lark. Therefore, a version of lark is downloaded that is not compatible with lark-parser~=0.7 and errors occurred.

---

In that case, 
```suggestion
    "lark>=0.7, <1.0"
```
should do.